### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]: On deleting the "Artifact" in Run on web dialog, the entire dialog becomes not accessible. 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1506]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Tab to and select created Project ->Azure->Run on web app.
4. "Run on web app" dialog will open.
5. Tab to and select the remove button, to delete the artifact,  then observe the keyboard behaviour.

**Expected Result:**
After deleting the "Artifact" the keyboard focus should stay on delete button and should be able to tab with tab key.

**Actual Result:**
No keyboard action is taking after deleting  the "Artifact"  in Run on web dialog.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![no keyboard action after deleting artifact in run on webapp](https://user-images.githubusercontent.com/22956130/36144532-28598f3c-1063-11e8-95f6-5872a5c3aea8.png)

**Comment by Eskibear** [comment]
duplicate with #1508

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749936
